api/v1alpha1/reaper_types.go

```diff
@@ -106,6 +112,15 @@ type ReaperSpec struct {
 
 	// Tolerations applied to the Reaper pods
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// InitContainerConfig encapsulates configs applied to the Reaper init container
+	InitContainerConfig InitContainerConfig `json:"initContainerConfig,omitempty" yaml:"initContainerConfig,omitempty"`
```

Comment:
I would consider renaming this or at least the name in the json annotation since a pod can have multiple init containers. We name the container `reaper-schema-init`. Maybe include something along those lines in the name?

Reply:
Good idea, that makes sense.  Gives us more flexibility.